acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/privacy/condition/PrivateContractVerifier.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.dsl.privacy.condition;
+
+public class PrivateContractVerifier {
+
+  public PrivateContractVerifier() {}
+
+  public ExpectValidPrivateContractDeployedReceipt validPrivateContractDeployed(
+      final String contractAddress, final String senderAddress) {
```

Comment:
nit: should these be of type "Address"?

Reply:
nope - the inputs here are from the contract wrapper which are web3j types